react-web/src/components/AvatarDropdown/AvatarDropdown.tsx

```diff
@@ -7,13 +7,25 @@ import Menu from "@mui/material/Menu";
 import MenuItem from "@mui/material/MenuItem";
 import Avatar from "@mui/material/Avatar";
 import "./AvatarDropDown.scss";
+import useLocalStorage from "hooks/useLocalStorage";
 
 const AvatarDropDown = () => {
   const navigate = useNavigate();
   const dispatch = useAppDispatch();
   const logOut = () => {
     dispatch(logout());
   };
+  const [, setIsLoggedIn] = useLocalStorage(
+    "isLoggedIn",
```

Comment:
i see a lot of these descriptors repeating all over the files (e.g. `isLoggedIn`) . I guess it's time to move them into a separate file with all constants defined

Reply:
@bogdan-manole Instead of directly accessing localStorage.getItem('isLoggedIn') within the header (for the menu items) and within the AvatarDropdown (for triggering logout functionality) , I tried to DRY this into a custom hook, and listened to the hook. It is just the changing variable that is being listened to here. I do not actually understand the need/suggestion to move it into a separate file. 

To more elaborate the workflow I had in mind here -
Hook is written for watching value changes to the data stored in the LS. Data in the LS is dependent on a few conditions such as fresh login, persisting user's login state on reload, change wallet etc. All of this cannot be handled by a mere constant as there are side-effects to the changes made to login state or details in login.